dev-docs/bidders/exadsadserver.md

```diff
@@ -0,0 +1,484 @@
+---
+layout: bidder
+title: EXADS
+description: EXADS Bidder Adapter
+pbjs: true
+biddercode: exadsadserver
+gvl_id: 1084
+tcfeu_supported: true
+media_types: banner, video, native
+gpp_sids: tcfeu
+safeframes_ok: false
+deals_supported: false
+floors_supported: true
+ortb_blocking_supported: true
+---
+
+### Example
+
+A very minimal RTB Banner example that shows how to use the EXADS adapter.
+The most important attributes are: **endpoint**, **fid** and **zoneId**. You can get them after configuring the zones.
+For more details about ad formats and parameters, read it in the next sections.
+
+```js
+
+adUnits = [
+  {
+    code: 'postbid_iframe', // the frame where to render the creative
+    mediaTypes: {
+      banner: {
+        sizes: [300, 250],
+      },
+    },
+    bids: [
+      {
+        bidder: 'exadsadserver',
+        params: {
+          zoneId: 12345,
+          fid: '829a896f011475d50da0d82cfdd1af8d9cdb07ff',
+          partner: 'ortb_2_4',
+          siteId: '123',
+          siteName: 'test.com',
+          country: 'IRL,
+          userIp: '0.0.0.0',
+          userId: '1234',
+          impressionId: impression_id.toString(),
+          mimes: ['image/jpg'],
+          endpoint: 'https://your-ad-network.com/rtb.php',
+        },
+      },
+    ],
+  },
+];
+```
+
+### Configuration
+
+#### General settings
+
+Use `setConfig` to instruct Prebid.js to initilize the exadsBidAdapter, as specified below.
+
+* Set "debug" as true if you need to read logs;
+* Set "gdprApplies" as true if you need to pass gdpr consent string;
+* The tcString is the iabtcf consent string for gdpr;
+* Uncomment the cache instruction if you need to configure a cache server (e.g. for instream video)
+
+```js
+pbjs.setConfig({
+  debug: false,
+  //cache: { url: "https://prebid.adnxs.com/pbc/v1/cache" },
+  consentManagement: {
+    gdpr: {
+      cmpApi: 'static',
+      timeout: 1000000,
+      defaultGdprScope: true,
+      consentData: {
+        getTCData: {
+          tcString: consentString,
+          gdprApplies: false, // set to true to pass the gdpr consent string
+        },
+      },
+    },
+  },
+});
+```
```

Comment:
This is prebid core related stuff and shouldn't be part of the adapter. The danger that this gets stale is pretty high

Reply:
Hi @muuki88, do you mean removing all information from row number 57 to 84?

Reply:
Removed all General settings section